src/other/pages/what-is-new/story.js

```diff
@@ -0,0 +1,83 @@
+import { html, render } from 'lit-html';
+import '../../../components/10-atoms/text';
+import '../../../components/10-atoms/heading';
+import styles from './index.scss';
+import contact from '../utils/contact-footer';
+
+const getFormattedGitCommitMessage = answerJson => {
+  const formattedMessage = answerJson.items[0].commit.message
+    .replace('Publish\n\n', '')
+    .replace(/- /g, '<br/>');
+  return formattedMessage;
+};
+
+const getDateFromGitCommit = answerJson => {
+  const formattedDate = new Date(answerJson.items[0].commit.author.date);
+  return formattedDate.toLocaleString(navigator.language);
+};
+
+export default {
+  title: 'What is new',
```

Comment:
The structure is not really cool: What is new > What is new. Maybe move "Whats new" to "Welcome" directory?

Reply:
done